pandas/core/arrays/datetimelike.py

```diff
@@ -101,6 +101,7 @@ class DatetimeLikeArrayMixin(OpsMixin, NDArrayBackedExtensionArray):
         _generate_range
     """
 
+    _infer_matches: Tuple[str, ...]
```

Comment:
can you add a 1-line comment to this one, as its not completely obvious (others if you think it would help)

Reply:
updated + greenish